src/styles/utils/variables.scss

```
@@ -0,0 +1,11 @@
+$sunday-margin: 606px;
+$cell-size: 100px;
+$gap: 1px;
+$days-and-margins: (
+"sat" 1 * ($cell-size + $gap),
+"fri" 2 * ($cell-size + $gap),
+"thu" 3 * ($cell-size + $gap),
+"wed" 4 * ($cell-size + $gap),
+"tue" 5 * ($cell-size + $gap),
+"mon" 6 * ($cell-size + $gap)
+);
```

it's better to leave like this and move your calculation to a loop
```suggestion
$days-and-margins: (
"sat" 1,
"fri" 2,
"thu" 3,
"wed" 4,
"tue" 5,
"mon" 6
);
```